pkg/scheduler/capabilities/dynamicresources/dynamicresources.go

```diff
@@ -0,0 +1,808 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
Is this part directly imported into the scheduler code? Or has it been modified and adapted (I don't know much about it)? If not, should it be marked in the code?

Reply:
I just changed the `New` to `NewDRAPlugin` and deleted `EventsToRegister` and some related codes with it, I think volcano doesn't need them right now. These codes are copied from the kubernetes repo, so the license needs to keep.

Reply:
Understand, if so, if the upstream code changes, do I need to make the same changes?

Reply:
Yes, before the `dynamicResources` struct changed to `DynamicResources`, we can only copy the whole codes related into volcano, otherwise we can't directly call those extension points. It may cause us to miss tracking some repaired codes, but if we can access the `DynamicResources` struct in v1.32, the problem will be solved.